### PR TITLE
add swedish personumber

### PR DESCRIPTION
**Pull request by klowdo**
This adds generation of the swedish personumber

### CODE REVIEW

File: Source/Bogus/Extensions/Sweden/ExtensionsForSweden.cs [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+{
+   /// <summary>
+   /// API extensions specific for a geographical location.
+   /// </summary>
+   public static class ExtensionsForSweden
+   {
+      /// <summary>
+      /// Swedish national identity number (personummer)
+      /// </summary>
+      public static string Personummer(this Person person)
```
Comment (bchavez) [inline Source/Bogus/Extensions/Sweden/ExtensionsForSweden.cs:15]: Should the name of this extension method be `Personnummer` with two `nn`s?
https://en.wikipedia.org/wiki/Personal_identity_number_(Sweden)

Comment (klowdo) [reply Source/Bogus/Extensions/Sweden/ExtensionsForSweden.cs:15]: You are absolutely right! thanks i'll fix

Comment (klowdo) [reply Source/Bogus/Extensions/Sweden/ExtensionsForSweden.cs]: fixed

**Comment by bchavez** [comment]
@klowdo, changes released in https://www.nuget.org/packages/Bogus/33.0.2
Thanks for your contribution!

**Pull request merged**